docs/user/config-file/examples/README

```diff
@@ -0,0 +1,12 @@
+README
+
+What is config-file/examples for?
+
+We are collecting all examples of configurations used in documentation and on
+the website here.
+
+We are putting examples into subfolders and calling them `.readthedocs.yaml`
+because each example might turn into a fully runnable example at some point.
```

Comment:
He, yeah, I commented something pretty similar in https://github.com/readthedocs/readthedocs.org/issues/10406#issuecomment-1584170855

Reply:
Alright, deleting :+1: I wanted to flag this intention because the reason for calling it for instance `examples/sphinx/.readthedocs.yaml` and not say `examples/sphinx.yaml` was to maintain a structure that someone can easily integrate and build.

I'll keep the structure for now and delete what I wrote in the README.

Reply:
That seems fine. I assumed the structure might be redundant without extra files, but perhaps there will be multiple examples per tool or something.